assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -59,6 +60,7 @@ function FramesLayer() {
       tabIndex="-1"
     >
       <FramesPageArea>
+        <CanvasOverlay />
```

Comment:
Not sure if `framesLayer` is the best place for placing this overlay, framesLayer is basically responsible for displaying element frames only (the selection borders, etc.). But perhaps it is, thoughts, @dvoytenko?

Reply:
This is why I wanted a code review. Really not sure where to put this overlay? Putting it elsewhere would mean positioning into the right place, that might be tricky. Love feedback on this.